.github/workflows/cargo.yml

```diff
@@ -42,6 +42,41 @@ jobs:
         with:
           command: test
 
+  check_and_test_windows:
+    name: Check and test
+    strategy:
+      matrix:
+        toolchain:
+          - 1.57.0 # min supported version (https://github.com/webrtc-rs/webrtc/#toolchain)
+          - stable
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: 📦 Cache cargo registry
+        uses: actions/cache@v3
+        with:
+          path: ~/.cargo/registry
+          key: ${{ runner.os }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-registry-
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.toolchain }}
+          profile: minimal
+          override: true
+      - name: Copy to C drive
+         run: cp D:\a C:\ -Recurse
+      # - name: 🏭 Cache dependencies
```

Comment:
Would we still not want the dependency caching? Presumably D:\ has enough space for it

Reply:
I want to see the build working without this complication first. I don't quite understand where the caching is doing its thing.

Reply:
But then for some reason I only get the coverage report. Why can't I have all the github actions? :'(